src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import './button.css'
+
+const Button = ({ name, styleForButton }) => {
+    return (
+        <div>
+            <button className={styleForButton}>
```

Comment:
You should define classes inside the Button component instead of passing them to the component. You should only use the prop to decide which class to use

Reply:
I don't understand this one. I defined classes in .css file, not in the Button component, is that the issue here? Because I used the prop to decide which class to use, as you said.